about.html

```diff
@@ -0,0 +1,165 @@
+    />
+    <script
+      src="https://kit.fontawesome.com/adf6e8ca6b.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Food/About</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav1">
+        <a class="nav-link" href="#">Logout</a>
+        <a class="nav-link" href="#" aria-label="twitter"
+          ><i id="twitter-icon" class="fa fa-twitter text-white"></i
+        ></a>
+        <a class="nav-link" href="#" aria-label="facebook"
+          ><i id="facebook-icon" class="fa fa-facebook text-white"></i
+        ></a>
+      </nav>
+      <nav class="nav2">
+        <div class="nav2Content">
+          <a id="bordered-link" class="nav-item" href="#">USE $ TO BUY</a>
+          <a class="nav-item" href="#">Contact Us</a>
+          <a class="nav-item" href="about.html">About</a>
+          <a class="nav-item" href="index.html">Home</a>
+        </div>
+        <div class="navLogo">
+          <p>LINDAFOOD</p>
+        </div>
+      </nav>
+      <button class="mobile-menu" type="button">
+        <img
+          src="https://img.icons8.com/ios/50/000000/menu--v4.png"
+          alt="menu"
+        />
+      </button>
+      <div class="mobile-menu-list" id="about-menu">
+        <a class="nav-item, nav-link" href="index.html">Home</a>
+        <a class="nav-item" href="about.html">About</a>
+        <a class="nav-item" href="#">Buy Now</a>
+      </div>
+    </header>
+    <section class="about">
+      <h1 class="aboutTitle">TASTE OUR<br />FOOD MADE IN DRC</h1>
+      <p class="abou-textBox">
+        pibus ornare, magna risus aliquet ipsum, in imperdiet ex arcu ac risus.
+        Cras porta aliquam porttitor. Nunc sed tortor ipsum. Cras suscipit ex
+        risus, sed consequat quam aliquet id. Cras hendrerit suscipit odio, sit
+        amet venenatis eros vestibulum consectetur. Integer at enim sagittis,
+        lacinia massa nec, condimentum ex. Cras porta aliquam porttitor. Nunc
+        sed tortor ipsum. Cras suscipit ex risus, sed consequat quam aliquet id.
+        Cras hendrerit suscipit odio, sit am
+      </p>
+      <p class="about-normal-text">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos
+        tenetur consectetur magni vo
+      </p>
+      <a id="about-link" href="#"
+        >lindafood.rdc@gmail.com
+        <img
+          src="https://img.icons8.com/external-those-icons-lineal-those-icons/10/000000/external-right-arrows-those-icons-lineal-those-icons-1.png"
+          alt="close"
+      /></a>
+    </section>
+    <section class="logo-section">
+      <h2 class="lecturerTitle">Our food saling international brand</h2>
```

Comment:
- [ ] It will be better if you center this title what do you think

![title](https://user-images.githubusercontent.com/80895497/153725131-bf05f192-3be7-492c-a5de-b4ab778048a8.PNG)